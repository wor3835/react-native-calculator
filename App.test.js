import React from "react";
import renderer from "react-test-renderer";
import connect from "./Main";
import store from "./App";
import performOp from "./operations";

const app = {};

test("App renders appropriately", () => {
  const tree = renderer.create(<connect />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Creates store", () => {
  const tree = renderer.create(<store />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Opperation loads correctly", () => {
  const tree = renderer.create(<performOp />).toJSON();
  expect(tree).toMatchSnapshot();
});
