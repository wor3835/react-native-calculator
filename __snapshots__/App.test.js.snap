// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders appropriately 1`] = `<connect />`;

exports[`Creates store 1`] = `<store />`;

exports[`Opperation loads correctly 1`] = `<performOp />`;
